cmd/podman/create.go

```diff
@@ -301,13 +301,26 @@ func isPortInPortBindings(pb map[nat.Port][]nat.PortBinding, port nat.Port) bool
 func exposedPorts(c *cli.Context, imageExposedPorts map[string]struct{}) (map[nat.Port]struct{}, map[nat.Port][]nat.PortBinding, error) {
 	// TODO Handle exposed ports from image
```

Comment:
Probably should remove this comment as we are not handling imageExposedPorts

Reply:
fixed